app/src/main/res/layout/activity_gde.xml

```diff
@@ -12,37 +14,40 @@
   ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
   ~ See the License for the specific language governing permissions and
   ~ limitations under the License.
-  -->
+-->
 
-<android.support.v4.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                                        android:orientation="vertical"
-                                        android:id="@+id/drawer"
-                                        android:layout_width="fill_parent"
-                                        android:layout_height="fill_parent">
+<android.support.v4.widget.DrawerLayout
+  android:id="@+id/drawer"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  android:layout_width="match_parent"
+  android:layout_height="match_parent"
+  android:orientation="vertical">
 
-    <LinearLayout
-            android:orientation="vertical"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent">
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
 
-        <com.viewpagerindicator.TitlePageIndicator
-                android:id="@+id/titles"
-                android:layout_height="wrap_content"
-                android:layout_width="fill_parent" />
+    <org.gdg.frisbee.android.view.SlidingTabLayout
+      android:id="@+id/sliding_tabs"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:background="@color/theme_primary" />
 
-        <android.support.v4.view.ViewPager
-                android:id="@+id/pager"
-                android:layout_height="wrap_content"
-                android:layout_width="fill_parent"></android.support.v4.view.ViewPager>
-    </LinearLayout>
+    <android.support.v4.view.ViewPager
+      android:id="@+id/pager"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content" />
+  </LinearLayout>
 
-    <ListView
-            android:id="@+id/left_drawer"
-            android:layout_width="240dp"
-            android:layout_height="match_parent"
-            android:layout_gravity="start"
-            android:choiceMode="singleChoice"
-            android:divider="@android:color/transparent"
-            android:dividerHeight="0dp"
-            android:background="#111" />
-</android.support.v4.widget.DrawerLayout>
\ No newline at end of file
+  <ListView
+    android:id="@+id/left_drawer"
+    android:layout_width="240dp"
+    android:layout_height="match_parent"
+    android:layout_gravity="start"
+    android:background="#111"
+    android:choiceMode="singleChoice"
+    android:divider="@android:color/transparent"
+    android:dividerHeight="0dp" />
+</android.support.v4.widget.DrawerLayout>
+    
```

Comment:
Strange that Android Studio shows me the newline.
Performing code fixes on the Oxford Tube on my way to Oxford City using the bus' free Wi-Fi!
Amazing!